packages/e2e/features/ui/components/authenticator/reset-password-angular.feature

```diff
@@ -1,27 +0,0 @@
-Feature: Reset Password
-
-  This will be used specifically for Angular users who have changed the translations for 'Send Code' for GH #1784.
```

Comment:
Is this translations for 'Send Code' done elsewhere? If not, should the Angular examples for reset-password be updated?

Reply:
Good point, this was a weird behavior for Angular. This PR fully removed "Send Code" translation from Angular reset-password, so I do not think we need to check that custom translation anymore because it's no longer an edge case.

I updated `reset-password` Angular example to drop reference to that legacy translation as well.